src/modal/modal.js

```diff
@@ -129,6 +151,33 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.stackedMap'])
         element.addClass(attrs.windowTopClass || '');
         scope.size = attrs.size;
 
+        // get topmost modal object
+        var bodyIsOverflowing;
+        var modal = $modalStack.getTop();
+        
+        // only when modal is attached to body
+        if(modal && modal.value && modal.value.appendTo === 'body'){
```

Comment:
Shouldn't `modal` exist if the `uib-modal-window` directive is used? Also shouldn't `modal.value' & `modal.value.appendTo` exist at all times?